modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -25,26 +24,23 @@ Do not use this image registry credentials file as the pull secret when you inst
 ====
 endif::restricted[]
 
-ifdef::restricted[]
-[WARNING]
```

Comment:
If I remember right, this warning is from some customer case, we intended to add it. But I can not remember exact bugs or issues now.